### DISCUSSION TITLE
pag出现闪屏问题

### DISCUSSION THREAD

**Discussion by gewanwana** [Tencent/libpag#1924] [Q&A] [answered]
pag用本地软件预览没有问题用https://pag.art/player.html网页检查会出现闪屏
[极简科技_本地生活_信息咨询_商务大气_hori16x9_10~20s_4.pag.zip](https://github.com/Tencent/libpag/files/13485528/_._._._hori16x9_10.20s_4.pag.zip)

**Comment by zhalice2011** [comment, marked as answer]
你好, 请先使用 4.1 版本

latest版本:  https://pagtest.zhalice.com/pages/simple.html
4.1.43版本:  https://pagtest.zhalice.com//pages-special-version/simple.html

闪屏问题修复后我会在这里再次回复你
